src/stores/GasPriceStore.js

```diff
@@ -17,7 +17,7 @@ class GasPriceStore {
       console.log(data)
       this.gasPrices = data;
     }).catch((e) => {
-      this.alertStore.pushError(e)
+      this.alertStore.pushError(`Gas price oracle is not available`)
```

Comment:
So, what is our suggestion if the user see this message? Wait for a minute and try again or what?

Reply:
The gas price oracle is fetched only when page is loaded, so message can be updated to also display suggestion to try to access the page again after a couple of minutes. Does it makes sense?

Reply:
Yes, it should be more informative.

Reply:
Thanks, fixed.